pubspec.yaml

```diff
@@ -9,7 +9,7 @@ description: >
   Requires the use of WebDriver remote server.
 homepage: https://github.com/google/webdriver.dart
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.9.0 <2.10.0'
```

Comment:
According to https://dart.dev/null-safety/migration-guide#migrating-by-hand, seems like they recommend 2.12.0-0 minimum constraint?

Reply:
Yup. I'm just trying to get everything synced to github first.

Then I'll create a PR to update everything on github. Get CI working. Then send the PR to google3

Reply:
Gotcha, approval for now.